src/main/java/com/algorand/sdkutils/RunAlgodV2Generator.java

```diff
@@ -0,0 +1,51 @@
+package com.algorand.sdkutils;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+
+import com.algorand.sdkutils.generators.Generator;
+import com.algorand.sdkutils.generators.Utils;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+
+public class RunAlgodV2Generator {
```

Comment:
Could you make sure the README is updated to remove the legacy codegen comments, and add instructions for using the new code generator?

Reply:
Added a section to the README file. 
As for removing the previous information, that will be part of the deprecation process.